.github/workflows/test_models_go_tests.yml

```diff
@@ -0,0 +1,117 @@
+name: Library Go tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  test_go:
+    strategy:
+      fail-fast: false
+      matrix:
+        library: [
+          # TestModels/dafny-dependencies/StandardLibrary, # This stores current Polymorph dependencies that all TestModels depend on
+          TestModels/Aggregate,
+          # TestModels/AggregateReferences,
+          TestModels/CodegenPatches,
```

Comment:
I'm not sure what this is. Do we need this enabled?

Reply:
Its passing so I enable it